controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -972,6 +974,14 @@ bool master_thread::handle_cmdu_1905_steering_completed_message(const std::strin
     return son_actions::send_cmdu_to_agent(src_mac, cmdu_tx, database);
 }
 
+bool master_thread::handle_cmdu_1905_client_capability_report_message(
+    const std::string &src_mac, ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    auto mid = cmdu_rx.getMessageId();
+    LOG(INFO) << "Received CLIENT_CAPABILITY_REPORT_MESSAGE, mid=" << std::hex << int(mid);
```

Comment:
Please also log the details so we can check it in test_flows.

Reply:
Client Capability Report TLV should include the frame body of the most recently received (Re)Association Request frame from this client (if it is an error scenario this field is omitted).
This part will be done in a separate PR.
So, for now the log message can contain if this is an error scenario or not, and I'll add a TODO comment to add the details later.